one-page/modules/ROOT/pages/frost-federation.adoc

```diff
@@ -17,7 +17,7 @@ https://blockstream.com/assets/downloads/pdf/liquid-whitepaper.pdf:[Liquid:
 A Bitcoin Sidechain]. FROST Federation is both publicly verifiable and
 and makes progress in the presence of Byzantine failures.
 
-The https://github.com/pool2win/frost-federation:[FROST Federation
+The https://github.com/pool2win/frost-federation[FROST Federation
```

Comment:
I think adoc prefers the colon I am fine with the change. I will merge locally and check.

https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/#anatomy